### PR TITLE
Prioritize venv with conda as secondary option in virtual env docs

### PR DESCRIPTION
**Pull request by DimedS**
## Description

This PR implements changes suggested by @astrojuanlu :

- Recommends `venv` as the primary solution for managing virtual environments, with `conda` as the secondary option.
- Removes collapsible menus from our documentation to simplify the process of finding commands for users.
- Removes all mentions of `pipenv` from our guide.

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: docs/source/get_started/install.md [hunk 1]
```diff
@@ -84,36 +45,41 @@ To exit the environment:
...
 
 ```bash
-pip install pipenv
+conda create --name kedro-environment python=3.10 -y
 ```
 
-Create a directory for working with your project and navigate to it. For example:
+The example below uses Python 3.10, and creates a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12) for your project, and you can name it anything you choose.
+
+The `conda` virtual environment is not dependent on your current working directory and can be activated from any directory:
```
Comment (noklam) [inline docs/source/get_started/install.md:60]: Is this conda or venv? I see conda instruction under venv section

File: docs/source/get_started/install.md [hunk 2]
```diff
@@ -13,54 +13,15 @@
...
-
-### How to create a new virtual environment without using `conda`
-
-Depending on your preferred Python installation, you can create virtual environments to work with Kedro using `venv` or `pipenv` instead of `conda`.
-
-<details>
-<summary><b>Click to expand instructions for <code>venv</code></b></summary>
-
-If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:
+The recommended approach. If you use Python 3, you should already have the `venv` module installed with the standard library. Create a directory for working with your project and navigate to it. For example:
```
Comment (noklam) [inline docs/source/get_started/install.md:24]: Is it missing some venv activate script? I expect something like `bin/activate`

File: docs/source/get_started/install.md (hunk 1 above)
Comment (DimedS) [reply docs/source/get_started/install.md:60]: maybe you should better have a look to final version on https://github.com/kedro-org/kedro/blob/make-pip-and-venv-installation-instructions-more-visible/docs/source/get_started/install.md not for a code difference, looks like it's hard to understand here. I see it's all right with sections

File: docs/source/get_started/install.md (hunk 2 above)
Comment (DimedS) [reply docs/source/get_started/install.md:24]: it is provided, I haven't change that part:
```
Activate this virtual environment:
source .venv/bin/activate # macOS / Linux
```

File: docs/source/get_started/install.md [hunk 3]
```diff
@@ -84,36 +45,41 @@ To exit the environment:
 ```bash
 deactivate
 ```
-</details>
 
-<details>
-<summary><b>Click to expand instructions for <code>pipenv</code></b></summary>
 
-Install `pipenv` as follows:
+### How to create a new virtual environment without using `venv`
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:50]: "without `venv`" implies that we're covering more options. I would just rename this to

```suggestion
### How to create a new virtual environment using `conda`
```

File: docs/source/get_started/install.md [hunk 4]
```diff
@@ -84,36 +45,41 @@ To exit the environment:
...
+
+ [The second, less popular option is to install Conda.](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) Than from your terminal:
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:52]: Maybe

```suggestion
 [Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:
```

File: docs/source/get_started/install.md [hunk 5]
```diff
@@ -84,36 +45,41 @@ To exit the environment:
 ```bash
 deactivate
 ```
-</details>
 
-<details>
-<summary><b>Click to expand instructions for <code>pipenv</code></b></summary>
```
[the remark this answers is not in the document]
Comment (DimedS) [reply docs/source/get_started/install.md:49]: I agree, fixed

File: docs/source/get_started/install.md [hunk 6]
```diff
@@ -13,57 +13,18 @@
 
 ## Create a virtual environment for your Kedro project
 
-We strongly recommend [installing `conda` as your virtual environment manager](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it.
+We strongly recommend utilizing `venv` as your virtual environment manager if you don't already use it.
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:16]:
```suggestion
We strongly recommend using `venv` as your virtual environment manager if you don't already use it.
```

("utilising" sounds unnecessarily complex?)

File: docs/source/get_started/install.md [hunk 7]
```diff
@@ -84,36 +45,41 @@ To exit the environment:
...
 ```
-</details>
 
-<details>
-<summary><b>Click to expand instructions for <code>pipenv</code></b></summary>
 
-Install `pipenv` as follows:
+### How to create a new virtual environment using `conda`
+
+ [Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:52]:
```suggestion
[Another popular option is to use Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/). After you install it, execute this from your terminal:
```

**Review by astrojuanlu — APPROVED** [review]
LGTM with nitpicks!

**Comment by astrojuanlu** [comment]
Thank you @DimedS !!

**Pull request merged**